share/spice/biblio_search/subtitle_content.handlebars

```diff
@@ -0,0 +1,4 @@
+{{#if isbn}}
```

Comment:
I will add many things here. I was just testing them to see if they come up properly.
It wasn't. Let's hope it does now. Then this will be better-looking